content/en/docs/collector/scaling.md

```diff
@@ -242,11 +242,11 @@ Collector. For instance, each Collector could be responsible for one Kubernetes
 namespace or specific labels on the workloads.
 
 Another way of scaling the Prometheus receiver is to use the
-[Target Allocator](https://github.com/open-telemetry/opentelemetry-operator#target-allocator):
-it’s an extra binary that can be deployed as part of the OpenTelemetry Operator
-and will split the share of Prometheus jobs for a given configuration across the
-cluster of Collectors using a consistent hashing algorithm. You can use a Custom
-Resource (CR) like the following to make use of the Target Allocator:
+[Target Allocator](/docs/kubernetes/operator/target-allocator/): it’s an extra
+binary that can be deployed as part of the OpenTelemetry Operator and will split
+the share of Prometheus jobs for a given configuration across the cluster of
+Collectors using a consistent hashing algorithm. You can use a Custom Resource
+(CR) like the following to make use of the Target Allocator:
```

Comment:
Minor edits for correctness:
* change `job` to `scrape target`. `job` has a specific meaning in the Prometheus ecosystem, and this could cause confusion.
* Remove the claim that a consistent hashing algorithm is used. This is one of the options, but not the default, and either way this seems like too much of a technical detail for a high-level overview.
```suggestion
[Target Allocator](/docs/kubernetes/operator/target-allocator/): it’s an extra
binary that can be deployed as part of the OpenTelemetry Operator and will distribute Prometheus scrape targets for a given configuration across the cluster of
Collectors. You can use a Custom Resource
(CR) like the following to make use of the Target Allocator:
```

Reply:
thank you @swiatekm-sumo